clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -353,6 +353,32 @@ export const privacyRequestApi = createApi({
       }),
       invalidatesTags: ["Notification"],
     }),
+    getStorageDetails: build.query<any, any>({
+      query: (params) => ({
+        url: `storage/default/${params.storage_type}`,
+      }),
+    }),
+    setActiveStorage: build.mutation<any, any>({
```

Comment:
`set` as a prefix for these methods doesn't really fit with our naming pattern for API calls (it makes them sound like state-setting methods). I think `create` makes sense for a `PUT` request, but `put` would also be ok.